projects/Mallard/src/components/article/article-header/article-kicker.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { View } from "react-native";
+import { useArticleAppearance } from 'src/theme/appearance'
+import { HeadlineKickerText } from 'src/components/styled-text';
+import { longReadHeaderStyles, newsHeaderStyles } from './article-header-styles';
+
+const dontDisplayKicker = ["Opinion"];
+
+export interface ArticleKickerProps {
+    kicker: string,
+    type?: 'news' | 'longRead'
+};
+
+export const ArticleKicker = ({
+    kicker,
+    type
+}: ArticleKickerProps) => {
+    if(dontDisplayKicker.includes(kicker)){
```

Comment:
i'm not entirely sure this is the right logic  – ie, if all opinion kickers will be "opinion". I would use the appearance for this (you can get it as a second parameter in the `useArticleAppearance` hook